templates/case/_case_show/_resident_and_rent_board_shareds.html.twig

```diff
@@ -29,20 +31,22 @@
                             <label>{% trans %}Telephone number{% endtrans %}</label>
                             <input type="text" class="form-control" value="{{ case.complainantPhone }}" readonly>
                         </div>
-                        <div class="form-group col-md-3">
-                            <label>{% trans %}CPR number{% endtrans %}</label>
-                            <div class="input-group mb-3">
-                                {# TODO: Input correct CPR when property exists #}
-                                <input type="text" class="form-control" value="{{ case.complainantCPR }}" readonly>
-                                {#  <div class="input-group-append">#}
-                                {# TODO: Show status depending on validation result #}
-                                {# Not yet validated #} <button class="btn btn-secondary" type="button" data-toggle="tooltip" data-placement="auto" title="{{ 'Validate CPR number'|trans([], 'case', 'da') }}"><i class="fas fa-check-circle"></i></button>
-                                {# Success <button class="btn btn-success" type="button" data-toggle="tooltip" data-placement="auto" title="CPR number is validated"><i class="fas fa-check-circle"></i></button>#}
-                                {# Warning <button class="btn btn-warning" type="button" data-toggle="tooltip" data-placement="auto" title="Warning could not validate"><i class="fas fa-exclamation-circle"></i></button>#}
-                                {# Failed <button class="btn btn-danger" type="button" data-toggle="tooltip" data-placement="auto" title="Error validating"><i class="fas fa-times-circle"></i></button>#}
-                                {#</div>   #}
+                        {% if not is_granted('ROLE_BOARD_MEMBER') %}
```

Comment:
Use `{% if is_granted('ROLE_CASEWORKER') %}`

Reply:
Fixed in 4bc1a7723936a2511ae7d457d4dfec56d37c7bf1.